### DISCUSSION TITLE
How do you make the app installable on browsers like Chrome?

### DISCUSSION THREAD

**Discussion by fybedev** [amitmerchant1990/notepad#21] [Q&A]
I tried code from Mozilla Developer Docs, but it won't work.
I'm kind of curious of how you did that.

**Comment by amitmerchant1990** [comment]
You can learn how to implement it in this article: https://www.amitmerchant.com/Building-Simple-Offline-Notepad-Using-Service-Worker/

**Comment by fybedev** [comment]
Yes, I know. I tried adding the manifest, and following the tutorial to its
end 4 times, but an install button would not show.

2022년 8월 23일 (화) 오후 10:49, Amit Merchant ***@***.***>님이 작성:
